ui/desktop/app/components/hidden-secret/index.js

```diff
@@ -1,5 +1,14 @@
 import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
 
 export default class HiddenSecretComponent extends Component {
   textMask = '■ ■ ■ ■ ■ ■ ■ ■ ■ ■ ■ ■';
+
+  @tracked secretVisible = false;
```

Comment:
Since the component is called `Hidden Secret Component`, consider renaming this variable to `isHidden` or `isSecretHidden`.